packages/components/atoms/f-link/src/components/Link.vue

```diff
@@ -1,45 +1,136 @@
 <template>
-    <div
-        :class="$style['o-link']"
-        data-test-id="link"
-    >
-        {{ copy.text }}
-    </div>
+    <span>
+        <a
+            :class="[
+                $style['o-link'], {
+                    [$style['o-link--bold']]: isBold,
+                    [$style['o-link--noDecoration']]: !hasTextDecoration,
+                    [$style['o-link--full']]: isFullWidth,
+                    [$style['o-link--noBreak']]: noLineBreak
+                }]"
+            data-test-id="link-component"
+            aria-describedby="description"
```

Comment:
Would this point to a non-existent element if an aria description is not passed?

Reply:
Fixed, Thank you!